src/lib/y2iscsi_client/authentication.rb

```diff
@@ -0,0 +1,137 @@
+require "yast"
+
+# |***************************************************************************
```

Comment:
What are those `|` characters here?

Reply:
I copied the header from other files in this same module. I don't think there is any meaning other than looking like a square. :-)

The only weird thing here is that the `require "yast"` is above the copyright header. Copy&paste error, I guess.